django/middleware/csrf.py

```diff
@@ -19,7 +19,7 @@
 logger = logging.getLogger('django.request')
 
 REASON_NO_REFERER = "Referer checking failed - no Referer."
-REASON_BAD_REFERER = "Referer checking failed - %s does not match %s."
+REASON_BAD_REFERER = "Referer checking failed - %s does not match any allowed hosts."
```

Comment:
Let's say "any trusted origins" rather than "any allowed hosts" here, to direct people towards the right setting rather than the wrong one.